src/js/_enqueues/vendor/farbtastic.js

```diff
@@ -3,6 +3,7 @@
  *
  * Licensed under the GPL license:
  *   http://www.gnu.org/licenses/gpl.html
+ * 
```

Comment:
* Note: Due to the in-activity of Fabrastic Github repo 
 * [https://github.com/mattfarina/farbtastic/issues]
 * and JQuery depreciate the following methods, 
 * replace new methods for old ones at WP milestone: 6.3
 * 
 * Details are described here: 
 * [https://core.trac.wordpress.org/ticket/57946]
 * a. bind()
 * b. unbind()
 * c. mousedown()